frontend/src/pages/AlertDetails/AlertHeader/ActionButtons/index.tsx

```diff
@@ -0,0 +1,64 @@
+import './actionButtons.styles.scss';
+
+import { Button, Divider, Dropdown, MenuProps, Tooltip } from 'antd';
+import { Copy, Ellipsis, PenLine, Trash2 } from 'lucide-react';
+import CopyToClipboard from 'periscope/components/CopyToClipboard';
+import React from 'react';
+
+const menu: MenuProps['items'] = [
+	{
+		key: 'rename-rule',
+		label: 'Rename',
```

Comment:
Let's avoid using the `component/index.tsx` approach altogether. It makes it really hard to debug and find components using global VS code search

Reply:
Create ActionButtons.tsx file and move all the content from this file to ActionButtons.tsx file
In index.tsx -> import ActionButtons.tsx and export default the component.

Reply:
@YounixM , @vikrantgupta25 , for the main components, I have used the `component/index.tsx` pattern, but for nested components inside the main component folder, I have used the component name as file.  I guess I had missed it for this component. 

@YounixM , would it be okay of I implement your suggestion for the top level components, and keep the child components as they are?